src/engine/naive_engine.cc

```diff
@@ -55,7 +55,7 @@ class NaiveEngine final : public Engine {
     std::vector<VarHandle> const_vars;
     std::vector<VarHandle> mutable_vars;
     FnProperty prop;
-    const char* opr_name;
+    std::string opr_name;
```

Comment:
Was this an issue blocking this feature? just curious

Reply:
This is a bugfix for naive engine. If my remember is correct, MXNet already had this fixed with different approach. So we can drop this change from this PR.

Reply:
I don't think it's merged yet. I'm ok with the change here.